packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -147,10 +147,15 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
   handleTabClick(
     event: React.MouseEvent<HTMLElement, MouseEvent>,
     eventKey: number | string,
-    tabContentRef: React.RefObject<any>
+    tabContentRef: React.RefObject<any>,
+    href?: string
   ) {
     // When tab is an achor tag with href, cancel navigation event
-    event.preventDefault();
+    if (href) {
+      event.preventDefault();
+      window.history.pushState(null, null, href);
```

Comment:
Why manipulate the history manually here? Omitting the `preventDefault()` call should allow the browser to navigate normally.

Reply:
You can see in the `Using the nav element` example that simply removing the `preventDefault` call as I see you've done in a separate PR results in tabs not being focused. This is the reason history is being utilized above when an `href` is in play.

Reply:
I agree now that it is a problem with the Patternfly docs.